packages/wallets/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './ProgressBar.scss';
+
+type TProps = {
+    step: number;
+    amount_of_steps: Array<string>;
+    setStep: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
```suggestion
    active_index: number;
    indexes: Array<number>;
    setActiveIndex: React.Dispatch<React.SetStateAction<number>>;
```

Reply:
Not sure if the props names are correct this way or not, can ignore it 😬

Reply:
This is okay. Generic names